src/Doctrine/Annotation/Tokens.php

```diff
@@ -146,26 +148,6 @@ public function getPreviousMeaningfulToken(int $index): ?int
         return $this->getMeaningfulTokenSibling($index, -1);
     }
 
-    /**
-     * Returns the index of the closest next token of the given type.
-     *
-     * @param string|string[] $type
```

Comment:
actual type is `int|int[]`, docblock is wrong here. but hey, the whole function is not actually in use!

same for others